Expecto.Tests/Main.fs

```diff
@@ -1,7 +1,20 @@
 ﻿module Main
 
 open Expecto
+open Expecto.Tests
+open Expecto.Impl
 
 [<EntryPoint>]
 let main args =
+  let localList = 
+    testList "local" [
+      ftestList "focused" [
+        testCase "test" (fun () -> ())
+      ]
+    ]
+
+  // check if we can fail on focused tests
+  if runTests { defaultConfig with failOnFocusedTests = true } localList <> 1 then
```

Comment:
Mhm I guess now we could have that. Will send ammendment or if you decide to merge then I will send another PR.